pkgs/development/libraries/protobuf/generic-v3-cmake.nix

```diff
@@ -90,6 +90,11 @@ let
     # unfortunately the shared libraries have yet to been patched by nix, thus tests will fail
     doCheck = false;
 
+    postInstall = ''
+      # pybind11_protobuf needs access to python/google/protobuf/proto_api.h
```

Comment:
let's avoid comments in nix strings. they unnecessarily affect the nix drv hash

Reply:
Ok.